_docs/_latest/setup/manual-deployment.md

```diff
@@ -51,24 +52,40 @@ service account, see
 
 * For information about the specifications needed for a Forseti server
   VM instance, see the [Compute Engine deployment template](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
-* When you create the instance, bind the server service account to the VM instance
+* When you create the instance, set the service account of the instance to the service account we just created or 
+if you already created the instance, you can [update the service account](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes).
 
 ### Install the Forseti Server
 
 To install the Forseti Server, follow the steps below:
 
 1. SSH into the server VM
 1. Run the following command to become an Ubuntu user:
-
-        sudo su ubuntu
-        
+    ```bash
+    sudo su ubuntu
+    ```
 1. `git clone` from the [forseti repo](https://github.com/GoogleCloudPlatform/forseti-security), and check out the [latest release](https://github.com/GoogleCloudPlatform/forseti-security/releases) by their tags.
-        
-        git clone https://github.com/GoogleCloudPlatform/forseti-security.git
-        git checkout tags/<tag_number>
-        
-1. Run the steps in the [startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
-1. Create [firewall rules](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/install/gcp/installer/forseti_server_installer.py)
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/forseti-security.git
+    git checkout tags/<tag_number>
+    ``` 
+1. Follow the setup instructions in the [startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/9a505061637913aa19c5b47bbf5ba507ce3bcebf/deployment-templates/compute-engine/server/forseti-instance-server.py#L109) 
```

Comment:
Will this link always work?

Reply:
Updated to use the stable branch reference in the url